resources/views/fashi/admin/category/index.blade.php

```diff
@@ -0,0 +1,168 @@
+@extends('admin_master')
+
+@section('content')
+<br>
+<div class="container">
+    <div class="row mb-2">
+        <div class="col-sm-6">
+            <h1>Category</h1>
+        </div>
+    </div>
+
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModalCreate">{{ trans('text.create_category') }}</button>
+    <br><br>
+
+    @if (session('message'))
+        <div class="alert alert-success" role="alert">
+            {{ session('message') }}
+        </div>
+    @endif
+
+    @error('name')
```

Comment:
```suggestion
    @error ('name')
```
bên dưới mà có lỗi này thì sửa tương tự nhé